tests/testthat/test-tokenization.R

```diff
@@ -12,118 +12,4 @@
-                       "##able", "[UNK]")
-  testthat::expect_identical(test_result, expected_result)
-})
-
-
-test_that("Tokenizers handle edge cases correctly", {
-  test_string <- "remove char: \ufffd "
-  vocab <- load_vocab(vocab_file = "vocab.txt")
-
-  b_tokenizer <- BasicTokenizer(TRUE)
-  test_result <- tokenize(b_tokenizer, text = test_string)
-  expected_result <- c("remove", "char", ":")
-  testthat::expect_identical(test_result, expected_result)
-
-  wp_tokenizer <- WordpieceTokenizer(vocab, max_input_chars_per_word = 4)
-  test_result <- tokenize(wp_tokenizer, text = "excessively long")
-  expected_result <- c("[UNK]", "long")
-  testthat::expect_identical(test_result, expected_result)
-})
-
-
-test_that("whitespace_tokenize splits a string on whitespace", {
-  test_string <- " some\ttext\nwith whitespace "
-  test_result <- whitespace_tokenize(test_string)
-  expected_result <- c("some", "text", "with", "whitespace")
-  testthat::expect_identical(test_result, expected_result)
-})
-
-
-
-test_that("strip_accents replaces accented chars with nearest equivalents", {
-  test_string <- "fa\u00E7ile"
-  test_result <- strip_accents(test_string)
-  expected_result <- "facile"
-  testthat::expect_identical(test_result, expected_result)
-})
-
-
-test_that("split_on_punc splits a string before and after punctuation chars", {
-  test_string <- "stop! don't touch that."
-  test_result <- split_on_punc(test_string)
-  expected_result <- c("stop", "!", " don", "'", "t touch that", ".")
-  testthat::expect_identical(test_result, expected_result)
-
-  test_string <- "!"
-  test_result <- split_on_punc(test_string)
-  expected_result <- c("!")
-  testthat::expect_identical(test_result, expected_result)
-})
-
-
-
-
-test_that("is_whitespace correctly classifies characters", {
-  # tests from BERT: tokenization_test.py
-  testthat::expect_true(is_whitespace(" "))
-  testthat::expect_true(is_whitespace("\t"))
-  testthat::expect_true(is_whitespace("\r"))
-  testthat::expect_true(is_whitespace("\n"))
-  testthat::expect_true(is_whitespace("\u00A0")) # non-breaking space
-
-  testthat::expect_false(is_whitespace("A"))
-  testthat::expect_false(is_whitespace("-"))
-})
-
-
-test_that("is_control correctly classifies characters", {
-  # tests from BERT: tokenization_test.py
-  testthat::expect_true(is_control("\u0005")) # 'Enquiry' control character
-
-  testthat::expect_false(is_control("A"))
-  testthat::expect_false(is_control(" "))
-  testthat::expect_false(is_control("\t"))
-  testthat::expect_false(is_control("\r"))
-})
-
-
-
-test_that("is_punctuation correctly classifies characters", {
-  # tests from BERT: tokenization_test.py
-  testthat::expect_true(is_punctuation("-"))
-  testthat::expect_true(is_punctuation("$"))
-  testthat::expect_true(is_punctuation("`"))
-  testthat::expect_true(is_punctuation("."))
-
-  testthat::expect_false(is_punctuation("A"))
-  testthat::expect_false(is_punctuation(" "))
-})
+# maybe new tests here?
```

Comment:
I'd say either add relevant tests here or delete this file (but maybe add a ticket either way).

Reply:
deleted file.